src/components/Form/TextArea/index.tsx

```diff
@@ -4,7 +4,7 @@ import { FieldError } from 'react-hook-form'
 
 interface Props {
   label: string
-  hint?: string
+  hint?: string | JSX.Element
```

Comment:
How come? Should hint only be text data?

Reply:
Because the hint now includes an anchor tag